### PR TITLE
Turkish Language Support

### PR DESCRIPTION
**Pull request by onurcankaraman**
I added the Turkish language to make the dashboars more understandable in Turkey. In this way, employees can monitor the machine's status, etc. can see in the local language.

# Description

It's not a merge I did against a problem. I just added the translation of the Turkish language. For this, I set up an else if structure between the existing if - else structure.

## Type of change

Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
I haven't done any testing.

**Test Configuration**:
* Kubernetes Version: 
* Helm Version:
* Hardware:

# Checklist:

- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding changes to the changelog (if needed).
- [ ] I have made corresponding notices to the upgrade instructions (if needed)

### CODE REVIEW

File: golang/pkg/datamodel/stateModelString.go [hunk 1]
```diff
@@ -9,7 +9,8 @@ type LanguageCode int
 
 const (
 	LanguageGerman  LanguageCode = 0
-	LanguageEnglish LanguageCode = 1
+	LanguageTurkish  LanguageCode = 1
```
Comment (JeremyTheocharis) [inline golang/pkg/datamodel/stateModelString.go:12]: Please keep 1 as the language code for english. turkish can get 2 :)

File: golang/pkg/datamodel/stateModelString.go [hunk 2]
```diff
@@ -90,6 +91,79 @@ func ConvertStateToString(state int, languageCode LanguageCode) (stateString str
 		default:
 			stateString = fmt.Sprintf("Unbekannter Zustand %d", state)
 		}
+	} else if languageCode == 1 { // TURKISH
```
Comment (JeremyTheocharis) [inline golang/pkg/datamodel/stateModelString.go:94]: Please keep 1 as the language code for english. turkish can get 2 :)

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]

File: golang/pkg/datamodel/stateModelString.go (hunk 2 above)
Comment (onurcankaraman) [reply golang/pkg/datamodel/stateModelString.go:94]: Ok, I'll update now :)

**Review by JeremyTheocharis — APPROVED** [review]

**Comment by JeremyTheocharis** [comment]
Nice! Let's wait until @Scarjit is back from holidays and then we will be probably able to merge it

**Review by Scarjit — APPROVED** [review]

**Pull request merged**